CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ## v1.0.0
 
+- 2023-08-10: Updated the adapter to align with preCICE V3. Updated the examples accordingly.
```

Comment:
This entry should not go in the v1.0.0 part of the CHANGELOG but above it, so in the **Not released yet** part.